### ISSUE TITLE
unable to manage members when previously denied members exis

### ISSUE THREAD

**Issue by mfrederickson** [concerto/concerto#1276]
```
ActionView::Template::Error (Object must be a Date, DateTime or Time object. ".previously_denied_members" given.):
    126: 
    127:         <% if @denied.size > 0 %>
    128:             <span onclick="toggle_prev_denied()"> 
    129:               <h3 class="default-padding"><%= l('.previously_denied_members') %> &nbsp;
```

**Comment by mfrederickson** [comment]
This may have a pretty major impact for users-- not sure if you want to release a new version with this fix.

**Comment by mfrederickson** [comment]
This was closed by PR #1278
